lib/kontrast/image_helper.rb

```diff
@@ -0,0 +1,49 @@
+require "RMagick"
+require "workers"
+
+module Kontrast
+
+    class ImageHelper
+        def initialize(img1_path, img2_path)
+            @img1 = Magick::Image.read(img1_path).first
+            @img2 = Magick::Image.read(img2_path).first
+            @path = Kontrast.path
+        end
+
+        def crop(width)
+            # Let's not do anything if the images are already the same size
+            return if @img1.rows == @img2.rows
+
+            # Get max height of both images
+            max_height = [@img1.rows, @img2.rows].max
+
+            # Crop
+            Workers.map([@img1, @img2]) do |image|
+                image.extent(width, max_height).write(image.filename)
+            end
+        end
+
+        # Uses the compare_channel function to highlight the differences between
+        # two images Docs:
+        # http://www.imagemagick.org/RMagick/doc/image1.html#compare_channel
+        def compare(output_dir, output_file_name)
+            begin
+                distortion_metric = Magick.const_get(Kontrast.configuration.distortion_metric)
+                diff = @img1.compare_channel(@img2, distortion_metric) do |options|
+                    options.highlight_color = Kontrast.configuration.highlight_color
+                    options.lowlight_color = Kontrast.configuration.lowlight_color
+                end
+
+                output_path = "#{Kontrast.path}/#{output_dir}"
+                FileUtils.mkdir_p(output_path) # Just in case
+                diff.first.write(File.join(output_path, output_file_name))
+
+                return diff
+            rescue Magick::ImageMagickError => e
+                puts "Error comparing images: #{e.message}"
+                # TODO: Explain that
+                return [1, 1]
```

Comment:
yeah, that was just temporary to match the response of `compare_channel`, we actually never use the first value of the diff, so I'll change this method return the last element of `diff` or `1` in this catch exception block